### PR TITLE
PMM-8282-add-tooltips-to-tls-options: add tooltips

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
**What this PR does / why we need it**:
It might not be clear to user which file goes to which form
image
**Which issue(s) this PR fixes**:
https://jira.percona.com/browse/PMM-8282

### CODE REVIEW

**Review by fabio-silva — CHANGES_REQUESTED** [review]
@solovevayaroslavna we now have `tooltipText` and others as props for our inputs. Please check `AddAlertRuleModal.tsx` to see it working.

**Review by fabio-silva — APPROVED** [review]

File: public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts [hunk 1]
```diff
@@ -37,6 +37,11 @@ export const Messages = {
         cluster: 'Cluster',
         customLabels: 'Custom labels',
       },
+      tooltips: {
+        tlsCA: 'Provide the CA certificate (example: ca.pem)',
```
[the remark this answers is not in the document]
Comment (tiagomotasantos) [reply public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts:41]: Here's an example of a better description: 
```suggestion
        tlsCA: 'Content of the trusted Certificate Authority (CA) file used to sign client certificates',
```

Pls apply the same principle to the other tooltips. All the information required is in the following doc: https://www.postgresql.org/docs/9.1/ssl-tcp.html

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts [hunk 2]
```diff
@@ -37,6 +37,12 @@ export const Messages = {
         cluster: 'Cluster',
         customLabels: 'Custom labels',
       },
+      tooltips: {
+        tlsCA: 'Content of the trusted Certificate Authority (CA) file used to sign client certificates',
+        tlsCertificateKey: 'Content of the client private key file',
+        tlsCertificate:
+          'Content of the trusted client certificate file signed by one of the trusted Certificate Authority (CA)',
```
Comment (tiagomotasantos) [inline public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts:44]:
```suggestion
          'Content of the client certificate file signed by one of the trusted certificate authorities (CAs)',
```

File: public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts [hunk 3]
```diff
@@ -37,6 +37,12 @@ export const Messages = {
         cluster: 'Cluster',
         customLabels: 'Custom labels',
       },
+      tooltips: {
+        tlsCA: 'Content of the trusted Certificate Authority (CA) file used to sign client certificates',
```
Comment (tiagomotasantos) [inline public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.messages.ts:41]:
```suggestion
        tlsCA: 'Content of the trusted certificate authority (CA) file used to sign client certificates',
```

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]
Only saw now that they don't use capital letters for CAs (my bad sorry). Other than that very good job with updating the tooltips 🚀

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**